### PR TITLE
Expand HTTPS provision warning

### LINKED ISSUE #4403: Launch Essentials Doc Update
Re: [Launch Essentials](https://pantheon.io/docs/guides/launch/domains/)

Priority: Med

## Issue Description:

for existing sites, avoid https interruption section, we don't document how long the challenge file is valid

## Suggested Resolution

Set expectations:
- Get challenge
- Complete challenge within 7 days (168 hours)
- Point DNS at Pantheon within 30 days

You will need to get a new challenge if you don't successfully complete it within the first 7 days.

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #4403 

## Effect
PR includes the following changes:
- Adds to warning callout the timeouts for challenge response and DNS.-

## Remaining Work
- [x] Review from @ari-gold
- [x] Copy Review

## Post Launch
To be completed by the docs team upon merge: 
- [x] Update Status Report
- [ ] Archive from **Done** in Waffle

### CODE REVIEW

**Comment by alexfornuto** [comment]
Additional review requested because the new copy conflicts with existing information:

>     We recommend leaving the challenge file in place until the migration is complete. That way, if DNS is not updated within the 7 day window provided by Let's Encrypt, the certificate can be re-issued.

Looking for confirmation on the DNS changeover timeframe.

**Comment by ari-gold** [comment]
@marktheunissen @kf6nux  Can you please review this change and confirm that the existing information Alex shared in his last comment should be removed?

**Comment by kf6nux** [comment]
@ari-gold, I agree the conflict Alex pointed out should be removed.

Thanks for checking for internal consistency @alexfornuto .

File: source/_docs/guides/launch/03-domains.md [hunk 1]
```diff
@@ -41,9 +41,14 @@ To pre-provision HTTPS, CAA records must either:
...
-<div class="alert alert-danger">
-<h4 class="info">Warning</h4>
-<p markdown="1">Skipping this step will result in service interruption for existing sites that require or expect HTTPS. If you skip this step, HTTPS will be available within an hour **after** DNS routes to Pantheon.</p>
+<div class="alert alert-danger" markdown="1">
+####Warning {.info}
+Skipping this step will result in service interruption for existing sites that require or expect HTTPS. If you skip this step, HTTPS will be available within an hour **after** DNS routes to Pantheon.
+
+Once you begin this process, you have:
+
+ - 7 days to complete the challenge response. After that, you must issue a new challenge.
```
Comment (ari-gold) [inline source/_docs/guides/launch/03-domains.md]: issue -> get ?

**Pull request merged**